parts/linux/cloud-init/artifacts/update-node-labels.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Updates Labels for Kubernetes node
+After=kubelet.service
+[Service]
+Restart=on-failure
+RestartSec=300
```

Comment:
since we do on failure, we can and we want to retry it sooner, So maybe 30seconds?

Reply:
Changed to 30 seconds